docs/ARCHITECTURE.md

```diff
@@ -50,6 +50,27 @@ service manager.
 
 BPF helpers written in C and used by PID 1 can be found under `src/core/bpf/`.
 
+### Implementing Directives
+
+The system and session manager supports a large number of directives. These can generally
+be configured in three ways:
+
+1) Via textual, INI-style configuration files called `units`
```

Comment:
hmm, markdown renders 1), 2), 3), … as 1., 2., 3., … anyway, hence maybe write it like that right-away?

Reply:
They are called `unit files` btw, not `units`. The `units` are then instantiated off them, so they are closely related, but foo@.service is a name of a unit file, not of a unit, for example.